katalog/kong-mesh/global-control-plane/patches/kong-mesh-admission-webhook-cainjector.yaml

```diff
@@ -0,0 +1,4 @@
+- op: add
+  path: /metadata/annotations
+  value:
+    cert-manager.io/inject-ca-from-secret: kong-mesh-tls-cert
```

Comment:
We should write in the readme that cert-manager is now a dependency

Reply:
I agree with you, but I would address the README issue in the "original" `kong-mesh` branch.